lib/honeycomb/integrations/rack.rb

```diff
@@ -13,8 +13,11 @@ module Rack
       ["HTTP_VERSION", "request.http_version"],
       ["HTTP_HOST", "request.host"],
       ["REMOTE_ADDR", "request.remote_addr"],
+      ["HTTP_ACCEPT", "request.header.accept"],
+      ["HTTP_CONTENT_TYPE", "request.header.content_type"],
       ["HTTP_USER_AGENT", "request.header.user_agent"],
       ["rack.url_scheme", "request.protocol"],
+      ["rack.url_scheme", "request.scheme"],
```

Comment:
do the other beelines have both `request.protocol` and `request.scheme`? Should we remove one of these?

Reply:
Most have `scheme` but not `protocol`. I agree that both is less than ideal - I'll remove `protocol`.

Reply:
Good catch, done!